config/config.go

```diff
@@ -282,6 +284,9 @@ func saveConfig(cfg *Config) *Config {
 			core.AutoComplete = true
 			core.Output = JSON
 		}
+		if !conf.Section(ini.DEFAULT_SECTION).HasKey("postrequest") { {
+			core.PostRequest = false
+		}
```

Comment:
@kevin-lii seems some confusion here, core here is not reflected from the defaultCoreConfig(), and core.PostRequest is false by default. possible to check the conf section has _postrequest_ key and it's value is true, to set core.PostRequest = true? Are you able to verify the GET & POST calls with this change?

Reply:
Sorry you are right. Apologies about the delay in fixing this